### PR TITLE
docs(0002): update metadata, generated files

### PR DESCRIPTION
**Pull request by mDuo13**

### CODE REVIEW

File: 0002-crypto-conditions/0002-crypto-conditions.md [hunk 1]
```diff
@@ -23,6 +23,19 @@ author:
...
+    name: Rome Reginelli
+    org: Ripple
+    street: 300 Montgomery Street
+    city: San Francisco
+    region: CA
+    code: 94104
+    country: US
+    phone: -----------------
+    email: rome@ripple.com
+    uri: http://www.ripple.com
```
Comment (justmoon) [inline 0002-crypto-conditions/0002-crypto-conditions.md]: `https` please, same goes for my link.

File: 0002-crypto-conditions/0002-crypto-conditions.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 coding: utf-8
 
 title: Crypto-Conditions
-docname: draft-thomas-crypto-conditions-00
+docname: draft-thomas-crypto-conditions-02
```
Comment (justmoon) [inline 0002-crypto-conditions/0002-crypto-conditions.md]: After `-00` comes `-01`, no?

**Comment by justmoon** [comment]
LGTM aside from minor comments.

File: 0002-crypto-conditions/0002-crypto-conditions.md (hunk 2 above)
Comment (mDuo13) [inline 0002-crypto-conditions/0002-crypto-conditions.md]: The version should not have been 00 to begin with. The version at https://interledger.org/five-bells-condition/spec.html is numbered 01 already.

Comment (justmoon) [inline 0002-crypto-conditions/0002-crypto-conditions.md]: The first version must be -00, which the IETF tracker helpfully forced me to abide by when I uploaded it. Ignore https://interledger.org/five-bells-condition/spec.html - that link is deprecated and was never authoritative. Internet Drafts are tracked by the IETF and the current version number is -00:

https://tools.ietf.org/html/draft-thomas-crypto-conditions-00

So the next version we upload has to be -01.

**Comment by justmoon** [comment]
LGTM

@mDuo13 You should be able to merge, please lmk if not. (Our repos have a policy that the submitter of a PR is always the one to merge it - unless they're not a committer and can't merge.)

**Pull request merged**